platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/AddressBookUtils.java

```diff
@@ -129,41 +129,34 @@ public static AddressBook parseAddressBookText(@NonNull final String addressBook
      * `nextAvailableNodeId` followed by a comma and then the node id.  The node id must be a positive integer greater
      * than all nodeIds in the address book.
      *
-     * @param nextAvailableNodeIdText the text to parse.
+     * @param nextNodeId the text to parse.
      * @return the parsed node id.
      * @throws ParseException if there is any problem with parsing the node id.
      */
     @NonNull
-    public static NodeId parseNextAvailableNodeId(@NonNull final String nextAvailableNodeIdText) throws ParseException {
-        Objects.requireNonNull(nextAvailableNodeIdText, "The nextAvailableNodeIdText must not be null.");
-        final String[] parts = nextAvailableNodeIdText.split(",");
+    public static NodeId parseNextNodeId(@NonNull final String nextNodeId) throws ParseException {
+        Objects.requireNonNull(nextNodeId, "The nextNodeId must not be null.");
+        final String[] parts = nextNodeId.split(",");
         if (parts.length != 2) {
             throw new ParseException(
-                    "The nextAvailableNodeIdText [%s] does not have exactly 2 comma separated parts."
-                            .formatted(nextAvailableNodeIdText),
-                    0);
+                    "The nextNodeId [%s] does not have exactly 2 comma separated parts.".formatted(nextNodeId), 0);
         }
         if (!parts[0].trim().equals(NEXT_NODE_ID_KEYWORD)) {
             throw new ParseException(
-                    "The nextAvailableNodeIdText [%s] does not start with the keyword `nextAvailableNodeId`."
-                            .formatted(nextAvailableNodeIdText),
+                    "The nextNodeId [%s] does not start with the keyword `nextAvailableNodeId`.".formatted(nextNodeId),
```

Comment:
You missed 3 instances of the string `nextAvailableNodeId`

Reply:
I am hanging my head in shame.  I thought I did a find in all files.

Reply:
There is a fundamental law of the universe where no matter how advanced your refactoring tool, changing the name of a variable is guaranteed to miss one or two old usages in documentation.